shell/artifacts/Social/Social.recipes

```diff
@@ -79,7 +79,7 @@ recipe
     input = posts
     stats = stats
     output = rankedPosts
-  List
+  PostList
```

Comment:
here you could still use `List`, because the `WordsScoreRanker` already outputs posts ranked by score, doesn't it?

Reply:
Similar to below, it does, but the order is not assured as it's basically a set.